http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/DiskServiceInterp.scala

```diff
@@ -193,7 +193,15 @@ class DiskServiceInterp[F[_]: Parallel](config: PersistentDiskConfig,
     for {
       ctx <- as.ask
       paramMap <- F.fromEither(processListParameters(params))
-      disks <- DiskServiceDbQueries.listDisks(paramMap._1, paramMap._2, cloudContext).transaction
+      creatorOnly =
+        // Support filtering by creator either by role=creator query string, or creator=<user email> label
+        if (
+          params
```

Comment:
can probably make this a utility function in `package.scala` so that it can be used in various places

sth like
```
def extractCreator(callerEmail: WorkbenchEmail, queryStrings: Map[String, String]): Option[WorkbenchEmail] =???
```

Reply:
agreed! Should I extract your logic from the listRuntimes endpoints to the helper as well? Would it make sense to fold this into the logic of `processListParameters` in that file, since that currently handles parsing out `includeDeleted` and `includeLabels`?

Reply:
>agreed! Should I extract your logic from the listRuntimes endpoints to the helper as well? 
sounds good...

>Would it make sense to fold this into the logic of processListParameters in that file, since that currently handles parsing out includeDeleted and includeLabels?
nah, I don't really like that function as is..let's not make it even more complicated